pyproject.toml

```diff
@@ -22,7 +22,9 @@ dependencies = [
     "djangorestframework >= 3.12",
     "requests",
     "drf-standardized-errors",
-    "celery"
+    "celery",
+    "fidelius @ git+https://github.com/dimagi/pyfidelius.git@ay/x509-pub-key",
```

Comment:
Consider pinning to a specific commit than a branch

Reply:
Noted!  This was added for testing purpose and is now not a part of this PR.